### PR TITLE
Update dependencies to latest

### PR DESCRIPTION
**Pull request by elideg**

### CODE REVIEW

File: src/polyfills.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
 /** Evergreen browsers require these. **/
 // Used for reflect-metadata in JIT. If you use AOT (and only Angular decorators), you can remove.
-import 'core-js/es7/reflect';
+// import 'core-js/es7/reflect';
```
Comment (Kolomonkey) [inline src/polyfills.ts:46]: why remove this?

File: src/typings.d.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // https://github.com/typings/typings
 // https://www.typescriptlang.org/docs/handbook/writing-declaration-files.html
 
-declare var module: { id: string };
-declare var require: any;
-declare var TweenMax: any;
-declare var {}: any;
+// declare var module: { id: string };
```
Comment (Kolomonkey) [inline src/typings.d.ts:5]: why remove this?

File: src/app/concepts/02-merging-streams/merging-streams.component.ts [hunk 3]
```diff
@@ -15,18 +15,18 @@ interface Coordinate {
     <button #right mat-raised-button color="accent">Move Right</button>
     <div class="container">
       <div #ball class="ball"
-           [style.left]="position.x + 'px'"
-           [style.top]="position.y + 'px'">
+           [style.left]="position?.x + 'px'"
```
Comment (Kolomonkey) [inline src/app/concepts/02-merging-streams/merging-streams.component.ts:18]: I don't believe the safe navigation operators are necessary

File: src/polyfills.ts (hunk 1 above)
Comment (elideg) [reply src/polyfills.ts:46]: Angular now does this under the hood so it's no longer necessary. It was giving me a compilation error.

File: src/typings.d.ts (hunk 2 above)
Comment (elideg) [reply src/typings.d.ts:5]: I don't see why this is necessary, these types are provided else ware. It was throwing a duplicate identifier error.

**Review by simpulton — APPROVED** [review]

**Pull request merged**